processing/src/main/java/org/apache/carbondata/processing/surrogatekeysgenerator/csvbased/CarbonCSVBasedSeqGenStep.java

```diff
@@ -1268,6 +1305,22 @@ else if(isComplexTypeColumn[j]) {
     return newArray;
   }
 
+  private void failDataLoad(Object[] row, int index, String columnName, int msrCount)
```

Comment:
there are multiple lines common in both methods, please pass the message or get the message from an another method which will provide the required message inorder to log based on the strategy.